templates/admin/org/list.tmpl

```diff
@@ -16,12 +16,12 @@
 			<table class="ui very basic striped table">
 				<thead>
 					<tr>
-						<th>ID</th>
-						<th>{{.i18n.Tr "admin.orgs.name"}}</th>
+						<th data-sortt="oldest,newest">ID</th>
+						<th data-sortt="alphabetically,reversealphabetically,true">{{.i18n.Tr "admin.orgs.name"}}</th>
```

Comment:
maybe also have three individual datasets for clarity. E.g. `data-sortt-$varname`.

Reply:
@silverwind Nice idear but will this complicate the js script?

Reply:
Not really, just destructure the names:

```
const {a, b, c} = node.dataset;